docs/students_and_interns.md

```diff
@@ -0,0 +1,31 @@
+* After you submit code, check the continuous integration build, which runs all the tests against your code. If there are failing tests, identify what went wrong and fix it if you can. (If you can't, ask for help.)
+* If you're trying to judge how to spend your time — whether to put in more work on this project, or start making contributions somewhere else to improve your chances — feel free to talk with us about it. We don't make any decisions before the final applications are complete, but we can give you a frank assessment of your chances.
+
+### What makes for a competitive application
+A competitive applicant will have completed a series of code contributions, starting from small and simple things, and moving to more complex and varied issues. The volume of your contributions is not the important part; we're not trying to pit applicants against each other to extract as much free work as possible, we're trying to understand what it would be like to work with you on a big project. We're looking for evidence of your technical and communication skills. Can you work effectively with JavaScript, Ruby, and CSS? Do you write efficient and understandable code? Do you ask good questions? Do you communicate clearly in your commit messages and your GitHub interactions? Do you follow the conventions of the codebase? We're also looking for evidence that you're becoming comfortable with the codebase. Do your later contributions show that you understand how different parts of the system work together, and how changes will affect user? Depending on the specific project, do you have additional skills that will be needed, such as visual design or user research? 
+
+A competitive applicant will also put together a project plan demonstrates an clear understanding of both project concept — what is it that the project aims to accomplish and why — and how to translate that project into technical tasks in the context of the Wiki Education Dashboard codebase. The project idea that we provide as a starting point may be very vague, and part of the process of preparing the application is to get as much of an understanding of that project idea as you can, and turn that understanding into a plan. (User research to learn more about exactly what Dashboard users need out of a particular project or feature, is often a good thing to incorporate into your project timeline; if the details of exactly what to build aren't clear from the project idea, that probably means there are many open questions that we don't yet have answers to.) We also welcome applicants to come up with their own project ideas — which is an especially good way to show your understanding of the big picture of the Dashboard and its role in the Wikimedia ecosystem.
+
```

Comment:
Is there an actual rubric we can point to for how we choose interns? Creating something semi-official might be useful / add clarity.